pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic
+from sympy.physics.mechanics import ReferenceFrame
+
+# local
+from ..bodies import Body, Ground
+from ..force import Force
+
+class TestBody():
+    def setup(self):
+        self.body = Body('body')
+
+    def test_body_init(self):
+        assert self.body.name == 'body'
+        assert self.body.parent is None
+        assert self.body.child is None
+        assert isinstance(self.body.masscenter, Point)
+        assert isinstance(self.body.frame, ReferenceFrame)
+        assert isinstance(self.body.mass, Symbol)
+        assert len(self.body.inertia) == 2
+        assert isinstance(self.body.inertia[0], Dyadic)
+        assert isinstance(self.body.inertia[1], Point)
+
+    def test_body_properties_value(self):
+        name = self.body.name
+        assert self.body.masscenter == Point(name + '_MassCenter')
+        assert self.body.frame == ReferenceFrame(name + '_ReferenceFrame')
+        assert self.body.mass == Symbol(name + '_Mass')
```

Comment:
This also begs the question as to whether each object should have a mathematical name and a human readable name. In SymPy things like this are handled by printing different representations of the objects.